### ISSUE TITLE
Command's parameters get their case lowered.

### ISSUE THREAD

**Issue by emptywee** [StackStorm/hubot-stackstorm#12]
Hello.

Commands sent to the hubot, which passes them later to stackstorm, get their case lowered. I am seeing that this is done to get a matching command, but this causes different issues later. E.g., passing a hostname to ansible. Ansible looks for the exact name in its inventory. I think this can be worked around by copying the command to a temp var, changing it to lower case and get a matching command from it, then treating the original command accordingly. I will try to implement it locally on my server.

Thanks.

**Comment by emptywee** [comment]
JFYI, seems to be working for me:
```js
  robot.respond(/(.+?)$/i, function(msg) {
    var command, result, command_name, format_string;

    command = msg.match[1]; //.toLowerCase();

    result = command_factory.getMatchingCommand(command.toLowerCase());
```

**Comment by manasdk** [comment]
@emptywee

Thanks for the report. Can you provide the command literal for which you noticed the issue?

**Comment by emptywee** [comment]
You are welcome.

Originally I was going to get a status of one of the hosts on the network. I had its hostname in uppercase in the ansible inventory file.

![Issue1](http://i.imgur.com/uXZTeVL.png)

So I got surpised, when I saw that.

I changed the action to just output the parameters and got this:

![Issue2](http://i.imgur.com/D12PRXL.png)

Hope this sheds some light on the original problem.

**Comment by manasdk** [comment]
Yes this clarifies. I am working on some changes to this repo will try to get this fixed as well.

**Comment by emptywee** [comment]
Thanks.

**Comment by manasdk** [comment]
Fixed in #17
